content/src/main/content/jcr_root/apps/acs-samples/angularjs-integration/clientlibs/footerLib/js.txt

```diff
@@ -0,0 +1,13 @@
+#base=js
+
+angular/core/main.js
+angular/core/baseController.js
+angular/core/routingService.js
+
+angular/partialControllers/homePartialController.js
```

Comment:
Partials are the piece of the page that is being loaded in dynamically
through AJAX via Angular routing.  The idea here was to group the
controllers that are being assigned by Angular routing together in one
place.  Unlike other controllers, you won't see these in the markup, only
in the routing configuration.  I can change it, but I didn't want readers
to be confused as to how these are assigned.  I do have a comment in the
top of each of those files, so maybe this is not necessary.

On Sun, Mar 8, 2015 at 8:41 AM, david g. <notifications@github.com> wrote:

> In
> content/src/main/content/jcr_root/apps/acs-samples/angularjs-integration/clientlibs/footerLib/js.txt
> <https://github.com/Adobe-Consulting-Services/acs-aem-samples/pull/26#discussion_r26006152>
> :
>
> > @@ -0,0 +1,13 @@
> > +#base=js
> > +
> > +angular/core/main.js
> > +angular/core/baseController.js
> > +angular/core/routingService.js
> > +
> > +angular/partialControllers/homePartialController.js
>
> Is "partialController" naming an angularJS convention? Are these any
> different from normal controllers (dont these just inherit from a base
> controller)
>
> Just seeing if we can reduce the number of points that have the potential
> to confuse (case in point; I am wondering why this isnt just named
> "controllers")
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Adobe-Consulting-Services/acs-aem-samples/pull/26/files#r26006152>
> .
>